src/core/js/libraries/jquery.a11y.js

```diff
@@ -583,14 +583,17 @@
             return true;
         }
 
+        function nativePreventScroll(event) {
+            return preventScroll($.event.fix(event));
```

Comment:
Could probably do with a comment here along the lines of what you've already said in the issue.
> I've also used an intermediate function to turn the native event object into a jquery event object - which the preventScroll function is expecting.